### ISSUE TITLE
provide support for xlsb

### ISSUE THREAD

**Issue by spedygiorgio** [hadley/readxl#71]
Thank you for your great package. Do you think it is possible to obtain support for xlsb files?

**Comment by hadley** [comment]
Probably not - it looks like a lot of work, and I think it's relatively uncommon.

**Comment by spedygiorgio** [comment]
Btw, it appears a package provides support for xlsb files, see [SO](http://stackoverflow.com/questions/33321505/in-r-languague-how-to-read-xlsb-files). Could it be investigated to "clone" this functionality also in readxl. It would help not to load many packages (and namespaces)...

**Comment by hadley** [comment]
@spedygiorgio no, because that requires Excel to be installed.

**Comment by jfunction** [comment]
I just wanted to note that I read in somedata using readxlsb recently and an unfortunate amount of time lapsed before I realised it was **reading some values incorrectly**. Strangely, the expected value was always negative and when added to the value as read by readxlsb gave 10737418.24 which has [no obvious meaning](https://www.binaryconvert.com/result_double.html?decimal=049048055051055052049056046050052). Also readxlsb read in other negative values just fine so I couldn't see a meaningful pattern. 

Anyway, all I'm saying is, be careful not to use that library without checking that it's working as expected. Lastly, in case anyone wants to attempt this (including maybe future me), the xlsb format is documented [here](https://docs.microsoft.com/en-us/openspecs/office_file_formats/ms-xlsb/acc8aa92-1f02-4167-99f5-84f9f676b95a). 

In my case I was able to just save the xlsb as xlsx (which was 9Mb as opposed to the 3Mb xlsb file).
